diff -ur mailsync-5.1.1/acinclude/ac_with_cclient.m4 mailsync-5.1.1-patched/acinclude/ac_with_cclient.m4
--- mailsync-5.1.1/acinclude/ac_with_cclient.m4	2003-02-04 12:43:45.000000000 +0000
+++ mailsync-5.1.1-patched/acinclude/ac_with_cclient.m4	2004-06-10 17:17:15.000000000 +0000
@@ -48,7 +48,7 @@
    dnl Looking for c-client libraries
    dnl
    AC_MSG_CHECKING([for c-client library])
-   for l in `eval "echo {${CCLIENTLOCATIONS}}{,/c-client,/lib/c-client,/lib}/{libc-client,libc-client4,c-client}{${SOP}}"` ; do
+   for l in `eval "echo {${CCLIENTLOCATIONS}}{,/c-client,/lib/c-client,/lib,/lib64}/{libc-client,libc-client4,c-client}{${SOP}}"` ; do
     if test -r "$l" ; then
      AC_MSG_RESULT([found ${l}])
      CCLIENT_LIBS="$l"
diff -ur mailsync-5.1.1/acinclude/ac_with_openssl.m4 mailsync-5.1.1-patched/acinclude/ac_with_openssl.m4
--- mailsync-5.1.1/acinclude/ac_with_openssl.m4	2003-01-23 22:02:35.000000000 +0000
+++ mailsync-5.1.1-patched/acinclude/ac_with_openssl.m4	2004-06-10 17:47:24.000000000 +0000
@@ -22,7 +22,7 @@
   ifelse([$2], , :, [$2])
  else
   AC_MSG_CHECKING([for libssl])
-  for p in `eval "echo {${OPENSSLLOCATIONS}}{/lib,}"` ; do
+  for p in `eval "echo {${OPENSSLLOCATIONS}}{/lib,/lib64,}"` ; do
    if test -r "${p}/libssl.a" -o -r "${p}/libssl.so" ; then
     OPENSSL_LDFLAGS="-L${p}"
     AC_MSG_RESULT([found in ${p}])
diff -ur mailsync-5.1.1/acinclude/ac_with_pam.m4 mailsync-5.1.1-patched/acinclude/ac_with_pam.m4
--- mailsync-5.1.1/acinclude/ac_with_pam.m4	2003-01-22 22:28:18.000000000 +0000
+++ mailsync-5.1.1-patched/acinclude/ac_with_pam.m4	2004-06-10 17:18:50.000000000 +0000
@@ -21,7 +21,7 @@
   ifelse([$2], , :, [$2])
  else
   AC_MSG_CHECKING([for pam library])
-  for p in `eval "echo {${PAMLOCATIONS}}{/lib,}"` ; do
+  for p in `eval "echo {${PAMLOCATIONS}}{/lib,/lib64,}"` ; do
    if test -r "${p}/libpam.a" -o -r "${p}/libpam.so" ; then
     PAM_LDFLAGS="-L${p}"
     AC_MSG_RESULT([found in ${p}])
