diff -u -ur mailsync-5.1.0/src/channel.cc ../mailsync-5.1.0/src/channel.cc
--- mailsync-5.1.0/src/channel.cc	2003-03-27 10:28:41.000000000 +0200
+++ mailsync-5.1.0/src/channel.cc	2004-02-18 20:00:31.000000000 +0200
@@ -7,6 +7,7 @@
 #include "msgstring.h"
 #include <flstring.h>
 #include "msgid.h"
+#include <assert.h>
 
 extern Passwd*     current_context_passwd;
 extern options_t options;
diff -u -ur mailsync-5.1.0/src/mailsync_main.cc ../mailsync-5.1.0/src/mailsync_main.cc
--- mailsync-5.1.0/src/mailsync_main.cc	2003-03-28 14:30:17.000000000 +0200
+++ mailsync-5.1.0/src/mailsync_main.cc	2004-02-18 18:34:23.000000000 +0200
@@ -76,6 +76,7 @@
 #include <ctype.h>
 #include <stdio.h>
 #include <errno.h>
+#include <assert.h>
 extern int errno;               // Just in case
 
 #include <string>
diff -u -ur mailsync-5.1.0/src/msgid.cc ../mailsync-5.1.0/src/msgid.cc
--- mailsync-5.1.0/src/msgid.cc	2003-03-27 11:53:55.000000000 +0200
+++ mailsync-5.1.0/src/msgid.cc	2004-02-18 20:50:45.000000000 +0200
@@ -5,6 +5,7 @@
 #include "msgid.h"
 #include "options.h"
 #include <c-client.h>
+#include <assert.h>
 
 //////////////////////////////////////////////////////////////////////////
 //
